woocommerce-services.php

```diff
@@ -361,16 +361,7 @@ public function load_textdomain() {
 		public function init() {
 			$this->load_dependencies();
 			add_action( 'admin_init', array( $this, 'admin_enqueue_scripts' ) );
-
-			if ( ! $this->check_jetpack_install() ) {
-				add_action( 'admin_init', array( $this, 'admin_jetpack_notice' ) );
-				return;
-			}
-
-			if ( ! WC_Connect_Options::get_option( 'tos_accepted', false ) ) {
-				add_action( 'admin_init', array( $this, 'admin_tos_notice' ) );
-				return;
-			}
+			add_action( 'admin_init', array( $this->nux, 'set_up_nux_notices' ) );
 
 			$this->schedule_service_schemas_fetch();
```

Comment:
No, I don't think it does. We were relying on the connection and TOS-related checks before scheduling the schema, which were removed in #1036.